packages/web/src/components/image-selection/ImageSelectionButton.js

```diff
@@ -65,7 +66,7 @@ const ImageSelectionButton = ({
       {includePopup ? (
         <>
           <Button
-            ref={anchorRef}
+            ref={anchorRefProp ? undefined : anchorRefInner}
```

Comment:
curious what this ref stuff is for?

Reply:
so remember how the artist popover was weirldy positioned relative to the artwork? the reason was is because the popover was anchored to the button that opens it, not the artwork itself, so i updated things to accept an anchorRef, and kept the inner one around for compatibility with other usages where we actually want to anchor to the button.

Reply:
ah interesting. sounds good